docs/user-guide/logging/overview.md

```diff
@@ -0,0 +1,107 @@
+
+In this document, you can find:
+
+* A basic demonstration of logging in Kubernetes using the standard output stream
+* A detailed description of the node logging architecture in Kubernetes
+* Guidance for implementing cluster-level logging in Kubernetes
+
+In the last section the assumption is that logging back-end is present inside or outside of the cluster. If you're not interested in having cluster-level logging, you might still find the description how logs are stored and handled on the node to be useful.
+
+## Basic logging in Kubernetes
+
+In this section, you can see an example of basic logging in Kubernetes that outputs data to the standard output stream. This demonstration uses a [pod specification](/docs/user-guide/logging/counter-pod.yaml) with a container that writes some text to standard output once per second.
+
+{% include code.html language="yaml" file="counter-pod.yaml" %}
+
+To run this pod, use the following command:
+
+```shell
+$ kubectl create -f counter-pod.yaml
+pod "counter" created
+```
+
+To fetch the logs, use the `kubectl logs` command, as follows
+
+```shell
+$ kubectl logs counter
+0: Tue Jun  2 21:37:31 UTC 2015
+1: Tue Jun  2 21:37:32 UTC 2015
+2: Tue Jun  2 21:37:33 UTC 2015
+3: Tue Jun  2 21:37:34 UTC 2015
+4: Tue Jun  2 21:37:35 UTC 2015
+5: Tue Jun  2 21:37:36 UTC 2015
+...
+```
+
+You can use `kubectl logs` to retrieve logs from a previous instantiation of a container with `--previous` flag, in case the container has crashed. If your pod has multiple containers, you should specify which container's logs you want to access by appending a container name to the command. See the [`kubectl logs` documentation](/docs/user-guide/kubectl/kubectl_logs) for more details.
+
+## Logging at the node level
+
+![Node level logging](/images/docs/user-guide/logging/logging-node-level.png)
+
+Everything a containerized application writes to `stdout` and `stderr` is handled and redirected somewhere by a container engine. For example, Docker container engine redirects those two streams to [a logging driver](https://docs.docker.com/engine/admin/logging/overview), which is configured in kubernetes to write to a file in json format.
+
+**Note:** Docker json logging driver treats each line as a separate message. When it's used, there is no direct support for multi-line messages out of the box, it should be handled on the logging agent level or higher.
```

Comment:
"**Note:** The Docker json logging driver treats each line as a separate message. When using the Docker logging driver, there is no direct support for multi-line messages. To do so, you'll need to handle these at the logging agent level or higher."